### DISCUSSION TITLE
Build without hashed output

### DISCUSSION THREAD

**Discussion by danielleroux** [facebook/docusaurus#7157] [General]
Is it possible to build docusaurus without hashed javascript output?

<img width="278" alt="image" src="https://user-images.githubusercontent.com/2144313/162899668-00b18cef-d8fd-44e4-82a4-dc60b6516063.png">

**Comment by slorber** [comment]
Hashed outputs is useful for aggressive CDN/browser caching, avoiding name conflicts and other things.

You can create a site plugin, implement `configureWebpack` and change whatever you want to our default config. This is at your own risk, we don't recommend it, and we won't support you doing so.

Our default Webpack output config is 

```js
    output: {
      pathinfo: false,
      path: outDir,
      filename: isProd ? 'assets/js/[name].[contenthash:8].js' : '[name].js',
      chunkFilename: isProd
        ? 'assets/js/[name].[contenthash:8].js'
        : '[name].js',
      publicPath: baseUrl,
      hashFunction: 'xxhash64',
    },
```
